src/main/java/bio/terra/cli/command/groups/ListUsers.java

```diff
@@ -1,39 +1,39 @@
 package bio.terra.cli.command.groups;
 
-import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Group;
 import bio.terra.cli.command.shared.BaseCommand;
 import bio.terra.cli.command.shared.options.Format;
-import bio.terra.cli.service.SamService;
+import bio.terra.cli.command.shared.options.GroupName;
+import bio.terra.cli.serialization.userfacing.UFGroupMember;
+import java.util.Comparator;
 import java.util.List;
+import java.util.stream.Collectors;
 import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the third-level "terra groups list-users" command. */
-@Command(name = "list-users", description = "List the users in a group with a given policy.")
+@Command(name = "list-users", description = "List the users in a group.")
 public class ListUsers extends BaseCommand {
-  @CommandLine.Parameters(index = "0", description = "The name of the group")
-  private String group;
-
-  @CommandLine.Option(
-      names = "--policy",
-      required = true,
-      description = "The name of the policy: ${COMPLETION-CANDIDATES}")
-  private SamService.GroupPolicy policy;
+  @CommandLine.Mixin GroupName groupNameOption;
 
   @CommandLine.Mixin Format formatOption;
```

Comment:
It would be handy for the user to have a global switch to set everything to JSON or text until further notice.

Reply:
Interesting suggestion! Have you seen something like that with other CLIs?